src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt

```diff
@@ -114,6 +170,21 @@ data class CreateUserRequest(
   val defaultCaseloadId: String? = null
 )
 
+data class EmailNotificationDto(
+  @Schema(description = "Username", example = "TEST_USER")
+  val username: String,
+
+  @Schema(description = "First name of the user", example = "John")
+  val firstName: String,
+
+  @Schema(description = "email of the user", example = "Smith@gov.uk")
+  val email: String?,
+
+  @Schema(description = "admin id who enabled user", example = "ADMIN_USR")
+  val admin: String
+
+)
```

Comment:
This class can probably live somewhere else as this Controller doesn't do anything with it

Reply:
Controller method (fun enableUserByUserId())returns this DTO

Reply:
Moved to Service